packages/frontend-test-utils/knip-report.md

```diff
@@ -1,2 +1,15 @@
 # Knip report
 
+## Unused dependencies (2)
+
+| Name                      | Location     | Severity |
+| :------------------------ | :----------- | :------- |
+| @backstage/version-bridge | package.json | error    |
+| zod                       | package.json | error    |
+
+## Unlisted dependencies (1)
+
+| Name               | Location                          | Severity |
+| :----------------- | :-------------------------------- | :------- |
+| @internal/frontend | src/app/createExtensionTester.tsx | error    |
```

Comment:
Would be neat to not list the internal ones. They are marked as such in package.json, which is a better way of detect them than by name.

Reply:
Yeah, I think this is something we need to discuss. This might be a topic to bring up with the knip folks upstream.

I wonder if a mono-repo and `"private": true,` in package.json would be enough of a signal. Though, looking at https://github.com/search?q=repo%3Awebpro-nl%2Fknip+private+path%3A%2F%5Epackages%5C%2Fknip%5C%2Ffixtures%5C%2F%2F&type=code it seems that that signal is already used in their test fixtures.

The 

```json
  "backstage": {
    "role": "web-library",
    "inline": true
  },
```

part would only be available to plugins, which need to be upstream. And I don't see a way to extend ignored dependencies from those. Another thing to check.

Should I just add something name-based for now?

Reply:
Yeah it's the Backstage specific "online" concept I meant indeed.

Maybe let's leave it as a future improvement. It's still a lot better than before either way.

Reply:
I went with a regexp on the name for now.